tasks/section5.yml

```diff
@@ -11,15 +11,17 @@
       - rule_5.1.1
 
 - name: "SCORED | 5.1.1 | PATCH | Ensure cron daemon is enabled"
-  command: /bin/true
+  service:
+      name: crond
+      enabled: yes
   tags:
       - level1
       - level2
       - patch
       - rule_5.1.1
 
 - name: "SCORED | 5.1.2 | AUDIT | Ensure permissions on /etc/crontab are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/crontab | egrep ".00 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*